src/radix_connect/interaction_id.rs

```diff
@@ -0,0 +1,41 @@
+use crate::prelude::*;
+
+uniffi::custom_newtype!(WalletInteractionId, String);
+
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+pub struct WalletInteractionId(pub String);
```

Comment:
btw is `WalletInteractionId` in fact a Uuid? If so we should change this to `pub struct WalletInteractionId(pub Uuid);`, see `profile_id.rs`! :)

Reply:
Oh @GhenadieVP btw, uppercase `D` please, `WalletInteractionID`, to follow the other types, e.g. `ProfileID`. 

I might have initially broken Rust conventions going with uppercase `D` ... but we ought to be consistent (and might change all to lowercase in another follow up PR...).